frontend/src/Home.jsx

```diff
@@ -2,40 +2,17 @@ import React, { Component } from 'react';
 import logo from './logo.png';
 import './App.css';
 
-const headers = new Headers({
-  Accept: 'application/json',
-  'Content-Type': 'application/json',
-});
-
 class Home extends Component {
-  componentWillMount() {
-    this.testBackend();
-  }
-
-  state = {
-    test: 'nothing.',
-  }
-
-  testBackend = async () => {
-    const test = await fetch('http://localhost:80/api/', {
-      method: 'GET',
-      headers,
-      cache: 'default',
-    });
-    const testJson = await test.json();
-    if (testJson && testJson.status) {
-      this.setState({ test: testJson.status });
-    }
-  }
-
+  // eslint-disable-next-line class-methods-use-this
   render() {
     return (
-      <header className='App-header'>
-        <img src={logo} className='App-logo' alt='logo' />
+      <div>
+        <h2>Home</h2>
         <p>
-          The backend says: {this.state.test}
+          Welcome to the Compotency Assessment Tool.
```

Comment:
Would it be possible to just add some space between this welcome message and the logo?
They are very close to each other.

![image](https://user-images.githubusercontent.com/23021242/51910955-9a2dab80-239e-11e9-9cf9-aa7f80fde4d5.png)